mappings/net/minecraft/server/world/EntityTrackingAudience.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5629 net/minecraft/server/world/EntityTrackingAudience
```

Comment:
So the mappings are going on an *adventure*?

Reply:
What? Don't understand